tools/gulp-extras.js

```diff
@@ -99,7 +100,24 @@ var checkImports = function() {
     });
 };
 
+var executeCommand = function(command, args, callback) {
+    var process = child_process.spawn(command, args);
+
+    process.stdout.on('data', function(data) {
+        console.log("" + data);
+    });
+
+    process.stderr.on('data', function(data) {
+        console.error("" + data);
+    });
+
+    process.on('exit', function(code) {
```

Comment:
But there is one on `child_process` which is what we are using here. :) Let me take care of that.